bokehjs/test/common/defaults.coffee

```diff
@@ -62,9 +62,17 @@ check_matching_defaults = (name, python_defaults, coffee_defaults) ->
       continue
 
     # special case for date time tickers, class hierarchy and attributes are handled differently
-    if name = "DatetimeTicker" and k = "tickers"
+    if name == "DatetimeTicker" and k == "tickers"
       continue
 
+    # special case for Title derived text properties
+    if name == "Title" and (k == "text_align" or k == "text_baseline")
+      continue
```

Comment:
I'm sure not using the 'text' mixin on the `Title` coffee model was the best idea. It might be better to explicitly enumerate them instead.